CHANGELOG.md

```diff
@@ -135,7 +150,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Service: Handle scaleset nodes that never register, such as nodes with instance-specific setup script failures.  [#518](https://github.com/microsoft/onefuzz/pull/518)
 
 ### Changed
-* Agent: Added stdout/stderr logging and clarifying context during failures to the `generic_analysis` task.  [#522](https://github.com/microsoft/onefuzz/pull/522)
+* Agent: Added STDOUT/STDERR logging and clarifying context during failures to the `generic_analysis` task.  [#522](https://github.com/microsoft/onefuzz/pull/522)
```

Comment:
FWIW, I think it's actually better to standardize on lowercase for `stderr`/`stdout`. Rationale: I suspect it's how most people write them, and its how they appear as global vars and in man pages. If you want to flag them as being special sorts of words (i.e. identifiers), backticks for code formatting might be better.